cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -991,7 +996,7 @@ protected Service createHeadlessService(List<ServicePort> ports) {
             service.getSpec().setIpFamilies(templateHeadlessServiceIpFamilies.stream().map(IpFamily::toValue).collect(Collectors.toList()));
         }
 
-        log.trace("Created headless service {}", service);
+        RECONCILIATION_LOGGER.trace(reconciliation, "Created headless service {}", service);
```

Comment:
All these trace levels ones where we're just recording that we did something successfully: Do we really need them? They seem very low value.